src/Service/Form/Submission/FormSubmissionService.php

```diff
@@ -141,4 +150,19 @@ public function generateMailBody(array $submissions): string
             return $this->twig->createTemplate('Error in body template: '.$e->getMessage())->render();
         }
     }
+
+    public function isSortable(): bool
+    {
+        return true;
+    }
+
+    public function getEntityName(): string
+    {
+        return 'formSubmission';
+    }
+
+    public function count($context = null): int
+    {
+        return \count($this->formSubmissionRepository->findFormSubmissions());
+    }
```

Comment:
You should implement a counter function inside the repository. Like this you are fetching all records from DB